blocks/test/fixtures/core__quote__style-1.serialized.html

```diff
@@ -1,4 +1,4 @@
 <!-- wp:quote -->
-<blockquote class="wp-block-quote">
+<blockquote class="wp-block-quote ">
```

Comment:
Hmm, this extra space at the end is unexpected. I think we might need to add Lodash's `_.compact` one way or another to prevent joining on empty classnames. What might be happening is if `emptyProps.className === ''`, then `[ 'wp-block-quote', '' ].join( ' ' ) === 'wp-block-quote '`. We could also handle this by avoiding the `typeof` check in favor of a general [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) check, which would also ignore the empty string.

Reply:
Well, if you could take a look at the latest commit. Before we overcomplicate things I just threw a .trim() in and solved the problem that way. :)